include/crow/json.h

```diff
@@ -1851,10 +1851,11 @@ namespace crow
                             } f_state;
                             char outbuf[128];
 #ifdef _MSC_VER
-                            sprintf_s(outbuf, sizeof(outbuf), "%f", v.num.d);
+#define MSC_COMPATIBLE_SPRINTF(BUFFER_PTR, FORMAT_PTR, VALUE) sprintf_s((BUFFER_PTR), 128, (FORMAT_PTR), DBL_DECIMAL_DIG, (VALUE))
```

Comment:
I would not reintroduce the macro here

Reply:
@gittiver Do you have any suggestion?

Reply:
leave the sprintf_s and sprintfs together with the new format string and don't introduce the macro for this one time use.

Reply:
Alright, there you go!